newIDE/app/src/UI/EditorMosaic/index.js

```diff
@@ -117,6 +119,17 @@ const renderMosaicWindowPreview = props => (
   </div>
 );
 
+const renderMosaicWindowToolbar = props => (
+  <div className="mosaic-window-toolbar" style={{ width: '100%' }}>
+    <div className="mosaic-window-title" title={props.tooltip}>
```

Comment:
If tooltip is not defined, will this show undefined as tooltip? If yes, we should either not define title, or force a tooltip to be defined (by not making `tooltip` optional)